keras-runner/src/main/kotlin/org/jetbrains/research/keras/runner/nn/model/sequential/Perceptron.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.research.keras.runner.nn.model.sequential
+
+import org.jetbrains.research.keras.runner.nn.layer.dense.DenseLayer
+import scientifik.kmath.linear.Point
+import scientifik.kmath.linear.asPoint
+
+open class Perceptron(
+    name: String,
+    override val layers: List<DenseLayer>,
+    batchInputShape: List<Int?>
+) : SequentialModel<List<Float>, Float>(name, layers, batchInputShape) {
+
+    override fun predict(input: List<Float>?): Float {
+        input ?: return 0.0f
```

Comment:
Is it true in general? probably we should have here `List<Float>` and work with null on call site